VAMobile/src/components/MessageAlert.tsx

```diff
@@ -53,22 +53,20 @@ const MessageAlert: FC<MessageAlertProps> = ({
 
   return hasValidationError ? (
     <Box mb={theme.dimensions.standardMarginBetween}>
-      <AlertBox
-        border={'error'}
-        title={t('secureMessaging.formMessage.weNeedMoreInfo')}
-        text={text}
-        titleRole={'header'}
+      <AlertWithHaptics
+        variant="error"
+        header={t('secureMessaging.formMessage.weNeedMoreInfo')}
+        description={text}
         scrollViewRef={scrollViewRef}
         focusOnError={focusOnError}>
         <VABulletList listOfText={bulletedListOfText} />
-      </AlertBox>
+      </AlertWithHaptics>
     </Box>
   ) : replyTriageError ? (
     <Box mb={theme.dimensions.standardMarginBetween}>
-      <AlertBox
-        border={'error'}
-        title={t('secureMessaging.sendError.title')}
-        titleRole={'header'}
+      <AlertWithHaptics
+        variant="error"
+        header={t('secureMessaging.sendError.title')}
         scrollViewRef={scrollViewRef}
         focusOnError={focusOnError}>
         <TextView variant="MobileBody" my={theme.dimensions.standardMarginBetween}>
```

Comment:
Could potentially be partially simplified moving this to the `description` prop to AlertWithHaptics.